platform/util/platform-util-unavailableurls/src/main/java/org/codice/ddf/platform/util/http/UnavailableUrls.java

```diff
@@ -116,17 +120,31 @@ public boolean contains(String url) {
         }
 
         /**
-         * Computes the next timeout, doesn't exceed MAX_TIMEOUT_SECONDS
+         * Computes the next timeout, doesn't exceed maximum internval
          */
         private void backoff() {
-            timeout = Math.min(timeout * INITIAL_TIMEOUT_SECONDS, MAX_TIMEOUT_SECONDS);
+            if (timeout >= getMaxRetryInterval() / 2) {
```

Comment:
Because the interval double, we check against `getMaxRetryInterval() / 2`. This prevents a timeout whose period is larger than MAX.